candig/server/static/src/custom_visualization.js

```diff
@@ -127,24 +128,26 @@ $("#table2").off("change").change(function() {
 
 $("#adv1_confirm").off('click').click(function() {
     document.getElementById("adv1").innerHTML = '<div class="loader_bar"></div>';
-    countRequest($("#table1").val(), [$("#key1").val()], datasetId).then(function(response) {
+    countRequest($("#table1").val(), [$("#key1").val()], datasetId).then(function(data) {
+        let response = JSON.parse(data)["results"][$("#table1").val()][0];
```

Comment:
This is changed because the `countRequest` returns full response.. so an additional step is needed here to parse out the results..